.changeset/tasty-spoons-double.md

```diff
@@ -0,0 +1,48 @@
+---
+'astro': minor
+---
+
+The `experimental.contentCollectionJsonSchema` feature introduced behind a flag in [v4.5.0](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#450) is no longer experimental and is available for general use.
+
+If you are working with collections of type `data`, Astro will auto-generate JSON schema files for your editor to get IntelliSense and type-checking. A separate file will be created for each data collection in your project based on your collections defined in `src/content/config.ts` using a library called [zod-to-json-schema](https://github.com/StefanTerdell/zod-to-json-schema#known-issues).
```

Comment:
```suggestion
If you are working with collections of type `data`, Astro can now auto-generate JSON schema files for your editor to get IntelliSense and type-checking. A separate file will be created for each data collection in your project based on your collections defined in `src/content/config.ts` using a library called [zod-to-json-schema](https://github.com/StefanTerdell/zod-to-json-schema#known-issues).
```
Just realizing that we say it "will" do something, but you have to do some manual steps. Is "can" more accurate?

And, I wouldn't put this in official documentation, but I do like using "now" for this changeset message, drawing attention to what's new.